services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx

```diff
@@ -21,6 +21,62 @@ import { Spinner } from '../Spinner/Spinner';
 import { EnvironmentGroup, ProductSummary } from '../../../api/api';
 import { useSearchParams } from 'react-router-dom';
 
+export type TableProps = {
+    productSummary: ProductSummary[];
+    teams: string[];
+};
+
+export const TableFiltered: React.FC<TableProps> = (props) => {
+    var versionToDisplay = (app: ProductSummary): string => {
```

Comment:
Please always try to use `const`. If that doesn't work, use `let`.
```suggestion
    const versionToDisplay = (app: ProductSummary): string => {
```